key_windows.go

```diff
@@ -0,0 +1,360 @@
+//go:build windows
```

Comment:
@aymanbagabas do we need this in v2?

Reply:
No, we don't. This logic is part of x/input